shell/browser/api/electron_api_power_monitor_win.cc

```diff
@@ -44,7 +43,12 @@ void PowerMonitor::InitPlatformSpecificMonitors() {
 
   // For Windows 8 and later, a new "connected standy" mode has been added and
   // we must explicitly register for its notifications.
-  if (base::win::GetVersion() >= base::win::Version::WIN8) {
```

Comment:
Curious, what caused this condition to not work on windows 7 ?

Reply:
C++ is not JavaScript :) The import table in the binary is fixed.

Reply:
<img width="1080" alt="Screen Shot 2020-08-27 at 6 55 48 PM" src="https://user-images.githubusercontent.com/1281234/91471904-f61c2700-e896-11ea-95f4-bdb610bd63b0.png">